app/models/solidus_affirm/gateway.rb

```diff
@@ -1,12 +1,13 @@
 module SolidusAffirm
-  class Gateway < Spree::Gateway
+  class Gateway < SolidusSupport.payment_method_parent_class
```

Comment:
Some day it would be great to rename this class into what is actually is (a `SolidusAffirm::PaymentMethod`), but this would force existing stores to change data on their database and therefor should be done in a mayor release instead.